packages/core/src/common/messaging/proxy-factory.ts

```diff
@@ -274,7 +278,7 @@ export class RpcProxyFactory<T extends object> implements ProxyHandler<T> {
         return p.toString().startsWith('notify') || p.toString().startsWith('on');
     }
 
-    protected serializeError(e: any): any {
+    protected serializeError(e: any, method?: string): any {
```

Comment:
I do like that improvement if there are actual cases where we get that parameter but I'm not seeing it. Do you have a case for that?